node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -90,4 +90,11 @@
 
 .vjs-tech {
     background-color: #000;
+    width: 98% !important;
+    left: 20px !important;
```

Comment:
It's better if we can avoid the `!important` statements. A better approach would be to put a `#filepreview-container` quantifier in front of out so the rules are limited to this widget and overrides happen correctly.